pkgs/development/libraries/flatbuffers/default.nix

```diff
@@ -3,33 +3,30 @@
 , fetchFromGitHub
 , fetchpatch
 , cmake
+, python3
 }:
 
 stdenv.mkDerivation rec {
   pname = "flatbuffers";
-  version = "2.0.0";
+  version = "2.0.6";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "flatbuffers";
     rev = "v${version}";
-    sha256 = "1zbf6bdpps8369r1ql00irxrp58jnalycc8jcapb8iqg654vlfz8";
+    sha256 = "sha256-0bJ0n/5yzj6lHXLKJzHUS0Bnlmys+X7pY/3LGapVh6k=";
   };
 
-  patches = [
-    # Pull patch pending upstream inclustion for gcc-12 support:
-    # https://github.com/google/flatbuffers/pull/6946
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Good catch! Proposed removal as https://github.com/NixOS/nixpkgs/pull/185281

Reply:
Damn it, I keep forgetting to check for that because the linter doesn't show any warnings (The ALE plugin in Vim runs `nix`). There should be some kind of tool support for this.